bootstrap.json

```diff
@@ -53,6 +54,10 @@
             "version": "v1.23.1",
             "cmd": "curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin $BOOTSTRAPPED_VERSION",
             "type": "shell"
+        },
+        "github.com/shuLhan/go-bindata/cmd/go-bindata" : {
+            "version": "master",
```

Comment:
I think we should use a tag, I believe at the time of this review `v3.4.0` is the latest.

Reply:
I Agree, I will check that everything is working with the latest tag and change it accordingly.